generation/core/generator.js

```diff
@@ -112,7 +129,7 @@ module.exports = function({
 		return json;
 	}
 
-	function createMethodBody(classJson, json) {
+	function createMethodBody(classJson, json, isTranslucent) {
```

Comment:
try not to pass booleans in function params, it's very confusing.
https://martinfowler.com/bliki/FlagArgument.html

Reply:
Instead, just create two functions with clear naming

Reply:
I moved this part down into the functions